app/assets/stylesheets/modules/_container.scss

```diff
@@ -0,0 +1,61 @@
+.drop-container{
+  position: absolute;
+  left: 50%;
+  margin-left: -570px;
+  top: 50%;
+  margin-top: -380px;
+  width: 1140px;
+}
+
+@media (max-height: 870px) {
+ .drop-container{
+    position: relative;
+    left: 0;
+    margin-left: 0;
+    top: 0;
+    margin-top: 0;
+  }
+}
+
+@media (max-width: 1140px) {
+ .drop-container{
```

Comment:
on line 10 starts a media query. It defines what the .drop-container class should behave like, if the maximum height of the container is 870px. In line 20 starts another media query that defines what the same class should behave like, if the container has a maximum height of 1140px. So the two media query don't overwrite each other, they are like two if statements for different cases.